library/core/src/option.rs

```diff
@@ -1564,10 +1501,6 @@ unsafe impl<A> TrustedLen for IterMut<'_, A> {}
 /// The iterator yields one value if the [`Option`] is a [`Some`], otherwise none.
 ///
 /// This `struct` is created by the [`Option::into_iter`] function.
```

Comment:
This is failing locally for me for some reason. Is it a bug?

```
error: unresolved link to `Option::into_iter`
    --> library/core/src/option.rs:1503:38
     |
1503 | /// This `struct` is created by the [`Option::into_iter`] function.
     |                                      ^^^^^^^^^^^^^^^^^^^ unresolved link
     |
     = note: `-D broken-intra-doc-links` implied by `-D warnings`
     = help: to escape `[` and `]` characters, add '\' before them like `\[` or `\]`

error: aborting due to previous error

error: Could not document `core`.
```

Reply:
It looks like it failed on CI as well.

Reply:
@camelid this is #74489

Reply:
Ah, so should I leave the old link then?

Reply:
Yes, at least until the other PR is merged.

Reply:
Okay, should be fixed now.